src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCompiler.cs

```diff
@@ -1013,7 +1013,6 @@ private void GenerateForwardSection()
          */
         private void GenerateMiddleSection()
         {
-            Label l1 = DefineLabel();
```

Comment:
@ViktorHofer, can you please review the Regex changes?

Reply:
I was on holiday and just saw this. From the docs: "This is just a token and does not yet represent any particular location within the stream.", https://docs.microsoft.com/en-us/dotnet/api/system.reflection.emit.ilgenerator.definelabel?view=netframework-4.8. This change should be fine.